src/pymodaq_plugins_keithley/hardware/keithley27XX/keithley27XX_VISADriver.py

```diff
@@ -279,18 +273,6 @@ def data(self):
 
         return str_answer, array_measurements_values, array_times_values
 
-    def get_card(self):
```

Comment:
in a base class, you should leave the method the real class should reimplement, like:
```
def get_card(self):
    """ Add a docstring """
    raise NotImplementedError
```